season/beta_challenges/hashes.md

```diff
@@ -0,0 +1,112 @@
+# Hashing It Out
+## Movies
+Manually create a hash with the following keys/values:
+| Key | Value |
+|---|---|
+| 'Hitchcock' | ['Rear Window']|
+| 'Truffaut' | ['400 Blows'] |
+| 'Myers'  | ['Austin Powers', 'Shrek']|
+| 'fruit' | 'banana' |
+Programmatically do the following:
+```ruby 
+random = {:Hitchcock=>["Rear window"], :Truffaut=>["400 blows"], :Myers=>["Austin Powers", "Shrek"], :Fruit=>"Banana"}
+```
```

Comment:
What's a better name for this hash? Also, make sure your keys are lower-cased.

Reply:
Good start, but I meant *all* the keys - including the ones that reflect proper nouns. Sorry for the nitpick!